Parsing error for "info files" when the path contains the string "is"

### Description
Parsing error for "info files" when the path contains the string "is"

```
pwndbg> heap
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /home/znl/pwndbg/pwndbg/commands/__init__.py:187 in __call__                                     │
│                                                                                                  │
│   184 │                                                                                          │
│   185 │   def __call__(self, *args: Any, **kwargs: Any) -> str | None:                           │
│   186 │   │   try:                                                                               │
│ ❱ 187 │   │   │   return self.function(*args, **kwargs)                                          │
│   188 │   │   except TypeError:                                                                  │
│   189 │   │   │   print(f"{self.function.__name__.strip()!r}: {self.function.__doc__.strip()}"   │
│   190 │   │   │   pwndbg.exception.handle(self.function.__name__)                                │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/commands/__init__.py:475 in _OnlyWithResolvedHeapSyms                    │
│                                                                                                  │
│   472 │   │   │   ):                                                                             │
│   473 │   │   │   │   # In auto mode, if the debug symbols are not enough, we will try to use    │
│   474 │   │   │   │   heuristic_heap = HeuristicHeap()                                           │
│ ❱ 475 │   │   │   │   if heuristic_heap.can_be_resolved():                                       │
│   476 │   │   │   │   │   pwndbg.aglib.heap.current = heuristic_heap                             │
│   477 │   │   │   │   │   w(                                                                     │
│   478 │   │   │   │   │   │   "pwndbg will try to resolve the heap symbols via heuristic now s   │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/aglib/heap/ptmalloc.py:1736 in can_be_resolved                           │
│                                                                                                  │
│   1733 │   │   return self._structs_module                                                       │
│   1734 │                                                                                         │
│   1735 │   def can_be_resolved(self) -> bool:                                                    │
│ ❱ 1736 │   │   return self.struct_module is not None                                             │
│   1737 │                                                                                         │
│   1738 │   @property                                                                             │
│   1739 │   def main_arena(self) -> Arena | None:                                                 │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/aglib/heap/ptmalloc.py:1726 in struct_module                             │
│                                                                                                  │
│   1723 │                                                                                         │
│   1724 │   @property                                                                             │
│   1725 │   def struct_module(self) -> types.ModuleType | None:                                   │
│ ❱ 1726 │   │   if not self._structs_module and pwndbg.glibc.get_version():                       │
│   1727 │   │   │   try:                                                                          │
│   1728 │   │   │   │   self._structs_module = importlib.reload(                                  │
│   1729 │   │   │   │   │   importlib.import_module("pwndbg.aglib.heap.structs")                  │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/aglib/proc.py:143 in wrapper                                             │
│                                                                                                  │
│   140 │   │   @functools.wraps(func)                                                             │
│   141 │   │   def wrapper(*a: P.args, **kw: P.kwargs) -> T | None:                               │
│   142 │   │   │   if self.alive:                                                                 │
│ ❱ 143 │   │   │   │   return func(*a, **kw)                                                      │
│   144 │   │   │   return None                                                                    │
│   145 │   │                                                                                      │
│   146 │   │   return wrapper                                                                     │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/glibc.py:62 in get_version                                               │
│                                                                                                  │
│    59                                                                                            │
│    60 @pwndbg.aglib.proc.OnlyWhenRunning                                                         │
│    61 def get_version() -> Tuple[int, ...] | None:                                               │
│ ❱  62 │   return cast(Union[Tuple[int, ...], None], glibc_version) or _get_version()             │
│    63                                                                                            │
│    64                                                                                            │
│    65 @pwndbg.aglib.proc.OnlyWhenRunning                                                         │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/aglib/proc.py:143 in wrapper                                             │
│                                                                                                  │
│   140 │   │   @functools.wraps(func)                                                             │
│   141 │   │   def wrapper(*a: P.args, **kw: P.kwargs) -> T | None:                               │
│   142 │   │   │   if self.alive:                                                                 │
│ ❱ 143 │   │   │   │   return func(*a, **kw)                                                      │
│   144 │   │   │   return None                                                                    │
│   145 │   │                                                                                      │
│   146 │   │   return wrapper                                                                     │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/lib/cache.py:162 in decorator                                            │
│                                                                                                  │
│   159 │   │   │   │   if cached_value is not _NOT_FOUND_IN_CACHE:                                │
│   160 │   │   │   │   │   return cached_value                                                    │
│   161 │   │   │   │                                                                              │
│ ❱ 162 │   │   │   │   value = func(*a, **kw)                                                     │
│   163 │   │   │   │                                                                              │
│   164 │   │   │   │   # Sanity check: its not perfect and won't cover all cases like ([],)       │
│   165 │   │   │   │   # but it should be good enough                                             │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/glibc.py:77 in _get_version                                              │
│                                                                                                  │
│    74 │   │   │   ver = pwndbg.aglib.memory.string(addr)                                         │
│    75 │   │   │   return tuple(int(_) for _ in ver.split(b"."))                                  │
│    76 │                                                                                          │
│ ❱  77 │   libc_filename = get_libc_filename_from_info_sharedlibrary()                            │
│    78 │   if not libc_filename:                                                                  │
│    79 │   │   return None                                                                        │
│    80 │   result = pwndbg.aglib.elf.dump_section_by_name(libc_filename, ".rodata", try_local_p   │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/aglib/proc.py:143 in wrapper                                             │
│                                                                                                  │
│   140 │   │   @functools.wraps(func)                                                             │
│   141 │   │   def wrapper(*a: P.args, **kw: P.kwargs) -> T | None:                               │
│   142 │   │   │   if self.alive:                                                                 │
│ ❱ 143 │   │   │   │   return func(*a, **kw)                                                      │
│   144 │   │   │   return None                                                                    │
│   145 │   │                                                                                      │
│   146 │   │   return wrapper                                                                     │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/lib/cache.py:162 in decorator                                            │
│                                                                                                  │
│   159 │   │   │   │   if cached_value is not _NOT_FOUND_IN_CACHE:                                │
│   160 │   │   │   │   │   return cached_value                                                    │
│   161 │   │   │   │                                                                              │
│ ❱ 162 │   │   │   │   value = func(*a, **kw)                                                     │
│   163 │   │   │   │                                                                              │
│   164 │   │   │   │   # Sanity check: its not perfect and won't cover all cases like ([],)       │
│   165 │   │   │   │   # but it should be good enough                                             │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/glibc.py:103 in get_libc_filename_from_info_sharedlibrary                │
│                                                                                                  │
│   100 │                                                                                          │
│   101 │   main_module_name = i.main_module_name()                                                │
│   102 │   seen = set()                                                                           │
│ ❱ 103 │   for address, size, sect_name, module_name in i.module_section_locations():             │
│   104 │   │   if module_name in seen:                                                            │
│   105 │   │   │   continue                                                                       │
│   106 │   │   seen.add(module_name)                                                              │
│                                                                                                  │
│ /home/znl/pwndbg/pwndbg/dbg/gdb.py:722 in module_section_locations                               │
│                                                                                                  │
│    719 │   │   │   div2 = div0[0].split("-", 1)                                                  │
│    720 │   │   │   # print(div0,div1,div2)                                                       │
│    721 │   │   │   assert (                                                                      │
│ ❱  722 │   │   │   │   len(div2) == 2                                                            │
│    723 │   │   │   ), "Wrong string format assumption while parsing the output of `info files`"  │
│    724 │   │   │                                                                                 │
│    725 │   │   │   beg = int(div2[0].strip(), 0)                                                 │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
AssertionError: Wrong string format assumption while parsing the output of `info files`
```

### Steps to reproduce

Gdb session history:
```
start
heap
```

### My setup
```
Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.35
OS: Ubuntu 22.04.4 LTS
OS ABI: #1 SMP Fri Mar 29 23:14:13 UTC 2024
Architecture: x86_64
Endian: little
Charset: utf-8
Width: 189
Height: 53
Gdb:      12.1
Python:   3.10.13 (main, Sep  5 2023, 06:03:44) [GCC 11.4.0]
Pwndbg:   2024.08.29 build: f2c51169
```

--- a/pwndbg/dbg/gdb.py
+++ b/pwndbg/dbg/gdb.py
@@ -741,21 +741,21 @@ class GDBProcess(pwndbg.dbg_mod.Process):
         result = []
         for line in files.splitlines():
             line = line.strip()
-            if "-" not in line or "is" not in line:
+            if " - " not in line or " is " not in line:
                 # Ignore non-location lines.
                 continue
 
-            div0 = line.split("is", 1)
+            div0 = line.split(" is ", 1)
             assert (
                 len(div0) == 2
             ), "Wrong string format assumption while parsing the output of `info files`"
 
-            div1 = div0[1].split("in", 1)
+            div1 = div0[1].split(" in ", 1)
             assert (
                 len(div1) == 1 or len(div1) == 2
             ), "Wrong string format assumption while parsing the output of `info files`"
 
-            div2 = div0[0].split("-", 1)
+            div2 = div0[0].split(" - ", 1)
             assert (
                 len(div2) == 2
             ), "Wrong string format assumption while parsing the output of `info files`"

